projects/mtg/include/MTGRules.h

```diff
@@ -58,6 +58,10 @@ class MTGEventBonus: public PermanentAbility
     int Angel[2];
     bool dragonbonusgranted[2];
     int dragon[2];
+	bool eldrazibonusgranted[2];
+	int eldrazi[2];
+	bool werewolfbonusgranted[2];
+	int werewolf[2];
```

Comment:
hope this screenshot will help, just click it to zoom
![screenshot 50](https://cloud.githubusercontent.com/assets/9781539/16636898/a629fe52-440d-11e6-8bb7-acf05540d08e.png)